packages/@aws-cdk/core/test/evaluate-cfn.ts

```diff
@@ -83,3 +94,15 @@ export function evaluateCFN(object: any, context: {[key: string]: string} = {}):
     return intrinsics[name].apply(intrinsics, args);
   }
 }
+
+function parseIntrinsic(x: any): { name: string; arg: any } | undefined {
+  if (typeof x !== 'object' || x === null) { return undefined; }
```

Comment:
Curious what spawned this new check. Presumably something broke. Worth a unit test?

Reply:
It didn't, really. I thought I needed it in 2 places, but turned out I didn't.

Left the code in because I think it reads more clearly.

Semi-specious argument: this is in test code by itself, and it is already tested by virtue of the existing tests still passing.